packages/devtools/devtools-testbench/src/style.css

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2022 DXOS.org
+ */
+
+/* Overriding kai/src/styles.css .full-screen to make kai render only for top half of the screen. */
+.full-screen {
```

Comment:
I think we can replace this with class declarations (here and in kai)

className='absolute left-0...`

Reply:
We probably should do it in next PR